tests/src/Common/CoreFX/CoreFX.depproj

```diff
@@ -36,7 +37,7 @@
       <Version>$(CoreFXXUnitPackageVersion)</Version>
     </PackageReference>
     <PackageReference Include="xunit.assert">
```

Comment:
Updating everything looks pretty non-trivial. So I am only updating the part that we need.